lib/msal-react/src/MsalProvider.tsx

```diff
@@ -86,6 +86,15 @@ export function MsalProvider({instance, children}: MsalProviderProps): React.Rea
         instance.handleRedirectPromise().catch(() => {
             // Errors should be handled by listening to the LOGIN_FAILURE event
             return;
+        }).finally(() => {
```

Comment:
We've had problems with ".finally" in the past with old browsers/polyfills, FYI.

Reply:
Is it a large enough concern to not use it here or can we address it if and when it comes up?